sdk/spring/spring-security-azure-oauth2-client/src/test/java/com/azure/spring/security/oauth2/client/ClientRegistrations.java

```diff
@@ -0,0 +1,70 @@
+package com.azure.spring.security.oauth2.client;
+
+import org.springframework.security.oauth2.client.registration.ClientRegistration;
+import org.springframework.security.oauth2.core.AuthorizationGrantType;
+import org.springframework.security.oauth2.core.oidc.IdTokenClaimNames;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static com.azure.spring.security.oauth2.client.OtherClientRefreshTokenOAuth2AuthorizedClientProvider.toTargetAuthorizationGrantType;
+
+public class ClientRegistrations {
```

Comment:
is this one new?

Reply:
Yes, it's new created in this PR.

Reply:
does this bring any new feature change?
I thought this PR is mainly about code structure only?

Reply:
> does this bring any new feature change?
 No.

> I thought this PR is mainly about code structure only?

I still add some unit tests and java docs in this PR.